base/siteorigin-widget.class.php

```diff
@@ -727,23 +727,34 @@ private function save_css( $instance ){
 
 		$css = $this->get_instance_css($instance);
 
-		if( !empty($css) ) {
-
+		if ( !empty( $css ) ) {
 			if ( WP_Filesystem() ) {
 				global $wp_filesystem;
 				$upload_dir = wp_upload_dir();
+				$add_cache = true;
```

Comment:
Is this variable necessary? I can't see a situation where this is anything but true?

Reply:
Whoops - it was for debugging.